src/plugins/data_sources/opensearch_dashboards.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "dataSource",
+  "version": "1.0.0",
+  "opensearchDashboardsVersion": "opensearchDashboards",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": ["data", "urlForwarding"],
```

Comment:
I think we will need to move this `data_source` plugin into data plugin? Data plugin includes index pattern definition, which means it will depends on data source, so having data source plugin to depend on data plugin will causes a circular dependency